src/foundry/foundry.js/clientDocuments/actor.d.ts

```diff
@@ -25,10 +27,267 @@ declare global {
    * ```
    */
   class Actor extends ClientDocumentMixin(foundry.documents.BaseActor) {
+    /**
+     * {@inheritdoc}
```

Comment:
I believe we decided to simple remove `@inheritdoc` because it works differently in tsdoc. also this seems to have been added by you. I would just remove it (and do the same with `@override` because that's also not present in foundry.js).